test/unit/specs/utils/isNumber.spec.js

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import isNumber from "../../../../src/utils/isNumber";
+
+describe("isNumber", () => {
+  it("tests", () => {
+    expect(isNumber(null)).toBe(false);
+    expect(isNumber(undefined)).toBe(false);
+    expect(isNumber("abc")).toBe(false);
+    expect(isNumber(parseInt("abc", 10))).toBe(false);
```

Comment:
This one is a bit odd. Since you're really just testing if `isNumber` returns `false` for `NaN`, how about just passing `NaN` into `isNumber` (ignore eslint if needed)?